step-generation/src/commandCreators/compound/transfer.ts

```diff
@@ -170,9 +170,11 @@ export const transfer: CommandCreator<TransferArgs> = (
     )
   }
 
-  const initialDestLabwareSlot = getSlotInLocationStack(
-    prevRobotState.labware[destLabware]?.stack
-  )
+  const initialDestLabwareSlot =
+    prevRobotState.labware[destLabware] != null
+      ? getSlotInLocationStack(prevRobotState.labware[destLabware]?.stack)
```

Comment:
I think we can omit the optional chaining since we are null checking first
```suggestion
      ? getSlotInLocationStack(prevRobotState.labware[destLabware].stack)
```